problemtools/verifyproblem.py

```diff
@@ -293,6 +284,15 @@ def get_subgroup(self, name):
         return next((sub for sub in self._items if isinstance(sub, TestCaseGroup) and os.path.basename(sub._datadir) == name), None)
 
 
+    def get_grader_flags_for_judging(self):
+        problem_type = self._problem.config.get('type')
+        if self.config['grading'] == 'default':
+            # The default grader flags are different depending on the problem type, which requires this override
+            default_grader_spec = ['always_accept', 'sum'] if problem_type == 'scoring' else ['no_errors', 'sum']
+            return default_grader_spec + self.config['grader_flags'].split()
```

Comment:
Looks good and I've been testing stuff, feel free to merge after this has been done.